src/app/zwiazek/miejsca-pamieci/miejsce/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+import { useParams } from "next/navigation";
+import { ComingSoon } from "@/components/ComingSoon/ComingSoon";
+
+export default function MemorialPlacePage() {
+    const { slug } = useParams();
```

Comment:
This doesn't seem to work.

To make it a [dynamic route](https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes) name the folder containing the page to `[slug]`.

Like this:
`./miejsce/page.tsx` -> `./[slug]/page.tsx`

Reply:
Other than that - good job on making room for future task of implementing this page.
You could add a Todo with the task link to indicate the intent of adjusting that and to make it easily searchable via task id 😉